manim/mobject/types/vectorized_mobject.py

```diff
@@ -2343,6 +2343,7 @@ def __init__(self, vmobject, **kwargs):
             part = VMobject()
             part.set_points(tup)
             part.match_style(vmobject)
+            self.append_points(tup)
             self.add(part)
```

Comment:
I have not investigated closely, but it seems like this change includes a certain redundancy when it comes to the points of `self`: not only are the points added to the mobject itself, but they are also added as part of `self`'s submobjects. This is not good (as the mobject would, in this case, also be drawn twice on the screen).

Reply:
Hey, thank you so much for the suggestions! I now see that implement ```CurvesAsSubmobjects```'s own method makes much more sense. I have updated the PR.

Thanks for your patience!